frontend/web/components/pages/GettingStartedPage.tsx

```diff
@@ -0,0 +1,359 @@
+  const { data: projects } = useGetProjectsQuery(
+    {
+      organisationId,
+    },
+    {
+      skip: !organisationId,
+    },
+  )
+  const project = projects?.[0]?.id
+  const { data: flags } = useGetProjectFlagsQuery(
+    {
+      project: `${project}`,
+    },
+    {
+      skip: !project,
+    },
+  )
+  const { data: environments } = useGetEnvironmentsQuery(
+    { projectId: `${project}` },
+    {
+      skip: !project,
+    },
+  )
+  const { data: segments } = useGetSegmentsQuery(
+    { projectId: `${project}` },
+    { skip: !project },
+  )
+  const environment = environments?.results?.[0]?.api_key
+  const items: GettingStartedItem[] = [
+    {
+      complete: !!projects?.length,
+      cta: 'Create Project',
+      description:
+        'Projects let you create and manage a set of features and configure them between multiple app environments',
+      duration: 1,
+      link: `/organisation/${organisationId}/projects`,
+      title: 'Create a Project',
+    },
+    {
+      complete: !!flags?.results?.length,
+      cta: 'Create Feature',
+      description:
+        'The first step to using feature flags is to create one in the features page',
+      disabledMessage: !project
+        ? 'You will need to create a project before creating your first feature'
+        : null,
+      duration: 1,
+      link: `/project/${project}/environment/${environment}/features`,
+      title: 'Create a Feature',
+    },
+    {
+      complete: !!segments?.results?.length,
+      cta: 'Create a Segment',
+      description:
+        'Once your features are setup you can target their rollout with segments',
+      disabledMessage: !project
+        ? 'You will need to create a project before creating your first segment'
+        : null,
+      duration: 1,
+      icon: 'pie-chart',
+      link: `/project/${project}/segments`,
+      title: 'Create a Segment',
+    },
+    {
+      description: 'Everything you need to get up-and-running with Flagsmith',
+      duration: 5,
+      link: 'https://docs.flagsmith.com/quickstart/',
+      persistId: 'quickstart',
+      title: 'Quick Start Guide',
+    },
+    {
+      description: "Familiarise yourself with Flagsmith's features",
+      duration: 1,
+      link: 'https://docs.flagsmith.com/basic-features/',
+      persistId: 'basic-features',
+      title: 'Feature Overview',
+    },
+    {
+      cta: 'Explore integrations',
+      description:
+        "Integrate Flagsmith with Flagsmith's features your existing tech stack",
+      duration: 1,
+      icon: 'layers',
+      link: 'https://docs.flagsmith.com/integrations/',
+      persistId: 'integrations',
+      title: 'Integrations',
+    },
+    {
+      complete: !!segments?.results?.length,
+      cta: 'Create a Segment',
+      description:
+        "Compare Flagsmith's free open source and commercial features",
+      duration: 1,
+      link: 'https://docs.flagsmith.com/version-comparison',
+      persistId: 'version-comparison',
```

Comment:
Would you be ok to rename this one in something like stepPersistedId/key to make it more self-explanatory?